repos/system_upgrade/common/actors/xfsinfoscanner/libraries/xfsinfoscanner.py

```diff
@@ -21,7 +21,13 @@ def scan_xfs_mount(data):
 
 
 def is_xfs_without_ftype(mp):
-    for l in run(['/usr/sbin/xfs_info', '{}'.format(mp)], split=True)['stdout']:
```

Comment:
@fernflower could add the check that the `mp` is really mountpoint? and in case it's not a mountpoint, just log info msg that particular entry is not mounted si skipping the check